administrator/components/com_workflow/Model/StateModel.php

```diff
@@ -320,7 +320,8 @@ public function publish(&$pks, $value = 1)
 				if ($table->load(array('id' => $pk)) && $table->default)
 				{
 					// Prune items that you can't change.
-					$app->enqueueMessage(\JText::_('COM_WORKFLOW_ITEM_MUST_PUBLISHED'), 'error');
+					$app->enqueueMessage(\JText::_('COM_WORKFLOW_MSG_DELETE_DEFAULT'), 'error');
```

Comment:
Does this language string exists?

Reply:
Yes ist does. Have a look at joomla-cms/administrator/language/en-GB/en-GB.com_workflow.ini line 51.